developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -387,6 +387,155 @@
     </Control>
   </Form>
 
+  <Form Name="context/options">
+    <Control Name="pages" Title="Options Dialog">
+      <p>The following options are configurable in TIKE in the General tab.</p>
```

Comment:
This line is probably not useful content to display for the pages control -- we need to write something that is helpful by itself without any other content. In this case, probably just describe the purpose of the Options dialog.

Reply:
Can I change it to something like this? 

"The Options dialog display several tabs such as General, Editor, Debugger, Character Map and Server. You can configure the environment of Keyman Developer here. "

Reply:
Yes, that sound fine